scripts/Inelastic/Direct/ReductionHelpers.py

```diff
@@ -228,10 +228,12 @@ def build_subst_dictionary(synonims_list=None) :
         if len(keys) < 2 :
             raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present")
         if len(keys[0]) == 0:
-            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, but the first key is empty")
+            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, "
```

Comment:
I think 7 characters are deducted for the second column of lime numbers when showing changes.